mep/pages/templates/pages/content_page.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
```

Comment:
Seems like it might make any sense to have a base content page template that all the other types extend.